### PR TITLE
an unatomic PR designed for maximizing maintainer suffering

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
- adds tinyfans to the nature shuttle. the one with the monkeys
- adds HAS_BONE and HAS_FLESH to all roundstart species (and ashwalkers) so that everyone is on an even playing field of getting their shit rocked by wounds
- above does not include slimepeople
- fucks around with the recipes for sutures/regen mesh by making them require sterile gauze
- fucks around with the sterile gauze recipe
- fucks around with bags in wardrobe vendors (adds duffels to sci, adjusts numbers, etc)
- esaber off icon state added
## Why It's Good For The Game
bone and flesh flags: wounds are cool and beating the shit out of people is a human right
tinyfans to nature shuttle: feature parity
suture/mesh/gauze recipes: underused imo
bags: bags are cool too. fucked up how there's no sci duffel, huh
esaber off icon: style but also fix
## Changelog
:cl:
add: The nature interaction shuttle with the monkeys now has tiny fans on the airlocks in, because that's apparently a feature that was missing.
add: More bags have been added to department vendors.
balance: Every roundstart species (and also ash walkers) now has flesh and bone that can be wounded.
balance: Recipes for sutures, regen mesh, and sterilized gauze have been adjusted to be easier, mostly.
balance: Sterilized gauze is better at absorbing blood and being a splint.
fix: Energy sabres now have an off inhand.
/:cl:

### CODE REVIEW

File: _maps/shuttles/emergency_nature.dmm [hunk 1]
```diff
@@ -413,8 +413,8 @@
 	pixel_y = 4
 	},
 /obj/item/reagent_containers/food/drinks/soda_cans/monkey_energy{
-	pixel_y = 5;
```
Comment (ghost) [inline _maps/shuttles/emergency_nature.dmm:416]: hello?? monkey man???????????

**Comment by timothyteakettle** [comment]
atomize it, slime HAS_FLESH should be there because they are entirely flesh, and should get cuts, just an oversight, so thanks for adding it in, thought you added HAS_BONE but checked again and you didn't, thank you very poggy

**Comment by DeltaFire15** [comment]
Not opposed to any of the changes currently in this PR.

**Review by DeltaFire15 — APPROVED** [review]
Atomize your stuff 
But, I don't exactly have issues with any of this, so I guess I'll just merge it.

**Pull request merged**